test/integration/testImage/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+REGISTRY=sonobuoy
+TARGET=testimage
+IMAGE_VERSION=v0.1
+
+build(){
+	docker build \
+    	-t $REGISTRY/$TARGET:$IMAGE_VERSION \
+    	-f Dockerfile \
+		.
+}
+
+build
```

Comment:
nit: newline EOF

Reply:
Thanks; used to have that setting but time away/updates seems to have removed it. Fixed.